README.md

```diff
@@ -191,7 +191,7 @@ Integration written by [Thanatos](https://github.com/ThanatosGit).
 ## Setup
 
 Getting started:
-   1. Clone this repository. (downloading the zip is *not* supported and will not work)
```

Comment:
Don't remove the zip mention. People press the download zip button way too often.

Reply:
Huh. Whenever people came into the rdv server for help, I never saw them had issues due to downloading the zip.

Reply:
Hopefully because both the script and the read me very explicitly call that out